uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -99,6 +99,28 @@ public static void installPackage(@NonNull final String packageName, @NonNull St
         Assert.assertEquals("Success", result.trim());
     }
 
+    /**
+     * Updates the supplied package on the device with the supplied flags.
+     *
+     * @param packageName the name of the package to update
+     * @param flags       the flags to use during for update of the app
+     */
+    public static void updatePackage(@NonNull final String packageName, @NonNull String... flags) {
```

Comment:
Looks like it is doing the same thing as the `installPackage` method. Why do we need this method?

Reply:
removed this, I added this method to add few differences but in the end removed them.